storage/badger/operation/approvals.go

```diff
@@ -1,31 +0,0 @@
-package operation
```

Comment:
This file gets moved to `storage/operation/approvals.go`, however git is not able to recognize it as a git file `rename`, still consider it as a deletion here, and there a new file addition.